src/System.Threading.Thread/pkg/System.Threading.Thread.pkgproj

```diff
@@ -1,14 +1,20 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Threading.Thread.csproj">
       <SupportedFramework>net463;netcoreapp1.1;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\src\System.Threading.Thread.builds" />
-    <NotSupportedOnTargetFramework Include="netcore50" />
   </ItemGroup>
   <ItemGroup>
+    <!-- Packages are now runtime-specific and they were not before, so this allows package verification during build to find
```

Comment:
> Packages are now runtime-specific [](start = 8, length = 34)

Small nit: The packages are not runtime-specific, but the actual implementation is, so I would change this comment to say something like: **Implementation is now runtime-specific** and it wasn't before, so this package allows.....